server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -141,6 +141,8 @@ private void createFateTable(ServerContext context) {
     } catch (Exception e) {
       if (e.getCause() instanceof KeeperException.NodeExistsException) {
         LOG.debug("Fate table node already exists in ZooKeeper");
+      } else {
+        throw e;
       }
```

Comment:
This change is looking for a very specific exception. There's no reason to check it with instanceof. It should just use a regular catch block:

```suggestion
    } catch (KeeperException.NodeExistsException e) {
      LOG.debug("Fate table node already exists in ZooKeeper");
```